### PR TITLE
Docs/qbittorrent qbitrr new 3rd party tool

### PR DESCRIPTION
**Pull request by Feramance**
# Pull Request

## Purpose

Update qBitrr doc to reflect recent updates

## Requirements

- [x] These changes meet the standards for [contributing](https://github.com/TRaSH-Guides/Guides/blob/master/CONTRIBUTING.md).
- [x] I have read the [code of conduct](https://github.com/TRaSH-Guides/Guides/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: docs/Downloaders/qBittorrent/3rd-party-tools.md [hunk 1]
```diff
@@ -21,30 +21,31 @@ This is a program used to manage your qBittorrent instance such as:
...
+   - `qbitrr DownloadedEpisodesScan` for Sonarr
 - Skip files in qBit entries by extension, folder or regex.
-- Monitor completed folder and cleans it up.
-- Uses ffprobe to ensure downloaded entries are valid media.
+- Monitor completed folder and clean it up.
+- Usage of [ffprobe](https://github.com/FFmpeg/FFmpeg) to ensure downloaded entries are valid media.
 - Trigger periodic Rss Syncs on the appropriate Arr instances.
 - Trigger Queue update on appropriate Arr instances.
-- Search requests from Overseerr or Ombi.
+- Search requests from [Overseerr](https://github.com/sct/overseerr) or [Ombi](https://github.com/Ombi-app/Ombi).
```
Comment (bakerboy448) [inline docs/Downloaders/qBittorrent/3rd-party-tools.md:39]: Add no referrer

**Review by bakerboy448 — DISMISSED** [review]

**Comment by TRaSH-** [comment]
> All done

Not really, you removed the link to ombi and overseerr.
just add after the link: `{:target="_blank" rel="noopener noreferrer"}`
this was when they click on the link it will open on a new page/tab etc

**Comment by Feramance** [comment]
Updated as explained

File: docs/Downloaders/qBittorrent/3rd-party-tools.md [hunk 2]
```diff
@@ -21,30 +21,31 @@ This is a program used to manage your qBittorrent instance such as:
...
 - Monitor qBit for Stalled/bad entries and delete them then blacklist them on Arrs (Option to also trigger a re-search action).
-- Monitor qBit for completed entries and tell the appropriate Arr instance to import it ( 'DownloadedMoviesScan' or 'DownloadedEpisodesScan' commands).
+- Monitor qBit for completed entries and tell the appropriate Arr instance to import it:
+   - `qbitrr DownloadedMoviesScan` for Radarr
+   - `qbitrr DownloadedEpisodesScan` for Sonarr
 - Skip files in qBit entries by extension, folder or regex.
-- Monitor completed folder and cleans it up.
-- Uses ffprobe to ensure downloaded entries are valid media.
+- Monitor completed folder and clean it up.
+- Usage of [ffprobe](https://github.com/FFmpeg/FFmpeg) to ensure downloaded entries are valid media.
```
Comment (bakerboy448) [inline docs/Downloaders/qBittorrent/3rd-party-tools.md:36]:
```suggestion
- Usage of [ffprobe](https://github.com/FFmpeg/FFmpeg){:target="_blank" rel="noopener noreferrer"} to ensure downloaded entries are valid media.
```
?

Comment (Feramance) [reply docs/Downloaders/qBittorrent/3rd-party-tools.md]: Fixed

**Comment by Feramance** [comment]
Should be all good now

**Review by bakerboy448 — CHANGES_REQUESTED** [review]
Sorry for the delay on this

**Review by bakerboy448 — APPROVED** [review]

**Pull request merged**